package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "voter-dapp",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/core": "^11.0.0",
```

Comment:
I think `@emotion/core` is not necessary here as it should be included by `@emotion/styled` and `@emotion/react`